schema/validation.py

```diff
@@ -0,0 +1,28 @@
+from jsonschema import validate
```

Comment:
It might be useful to reuse the mechanisms in OCR-D/core for this, `JsonValidator`, `click` CLI and bundling the resources with the python package. But we can do this later.